src/get_log.py

```diff
@@ -0,0 +1,22 @@
+import json
```

Comment:
What is the purpose of this whole script?

Reply:
Consolidated files